apps/saleor-app-checkout/pages/api/register.ts

```diff
@@ -1,22 +1,9 @@
-import { getClient } from "@/saleor-app-checkout/backend/client";
 import { NextApiRequest, NextApiResponse } from "next";
 
 import { saleorDomainHeader } from "../../constants";
-import { print } from "graphql/language/printer.js";
-import {
-  CheckWebhooksDocument,
-  CheckWebhooksQuery,
-  CheckWebhooksQueryVariables,
-  CreateWebhooksDocument,
-  CreateWebhooksMutation,
-  CreateWebhooksMutationVariables,
-  TransactionActionRequestSubscriptionDocument,
-} from "@/saleor-app-checkout/graphql";
-import { getBaseUrl } from "@/saleor-app-checkout/backend/utils";
-import { SALEOR_WEBHOOK_TRANSACTION_ENDPOINT } from "./webhooks/saleor/transaction-action-request";
 import { getAppDomain, setAuthToken } from "@/saleor-app-checkout/backend/environment";
-import urlJoin from "url-join";
 
+// eslint-disable-next-line require-await
```

Comment:
Hmmm... why do we want to disable that rule? If the route doesn't have any `await` then I think we can remove `async`?